BotVentic/BotVentic/MessageHandler.cs

```diff
@@ -322,6 +324,20 @@ private static string GetEmoteUrl(EmoteInfo emote_info)
                         Console.WriteLine(ex.ToString());
                     }
                     break;
+                case "!foodporn":
+                    try
+                    {
+                  
+                        string downloadString = await Program.RequestAsync("http://foodporndaily.com");
+                        string regexImgSrc = @"<img[^>]*?src\s*=\s*[""']?([^'"" >]+?)[ '""][^>]*?>";
+                        MatchCollection matchesImgSrc = Regex.Matches(downloadString, regexImgSrc, RegexOptions.IgnoreCase | RegexOptions.Singleline);
+                        reply = matchesImgSrc[1].Groups[1].Value;
+                    }
+                    catch
+                    {
+                        Console.WriteLine("Failed to get daily foodporn image");
```

Comment:
It should also reply to the command on Discord with an error message. That way if it breaks people will know if it's actually broken and not just temporarily down. Console would benefit from the actual exception details.